.changeset/nice-houses-sneeze.md

```diff
@@ -0,0 +1,24 @@
+---
+"@logto/core": minor
+---
+
+feat: enhanced user lookup by phone with phone number normalization.
```

Comment:
Should be either a fix or refactor, IMO. Also the trailing period `.` is not necessary